app/labor/markdown_fixer.rb

```diff
@@ -36,6 +36,12 @@ def modify_hr_tags(markdown)
       end
     end
 
+    def lowercase_published(markdown)
```

Comment:
Suggestion: are any front-matter tags allowed to be capitalized? Should we lowercase ALL of them?

Reply:
Yeah, I would second this. Just thinking that, unless we have a good enough reason not to, we may as well be consistent in terms of what we allow through markdown (seems like we're heading in the direction of being more forgiving re: case).